mappings/net/minecraft/block/Block.mapping

```diff
@@ -135,6 +135,7 @@ CLASS none/akw net/minecraft/block/Block
 	METHOD a getRawIdFromBlock (Lnone/akw;)I
 		ARG 0 block
 	METHOD a (Lnone/akw;Lnone/akw;)Z
+	METHOD a setSounds (Lnone/apu;)Lnone/akw;
```

Comment:
Are you sure "setSounds" is good? I don't like it. Perhaps "setSoundType"? "setSoundSet"? @shadowfacts